### ISSUE TITLE
Register Domain Name

### ISSUE THREAD

**Issue by msarmi9** [msarmi9/Sparkle#131]
Wherever we end up hosting our web app (elasticbeanstalk or heroku), it'd be nice to have a custom domain name like `sparkle.ai`.

**Comment by msarmi9** [comment]
Done! At long last, we have a domain: www.sparklemed.com. Setting up HTTPS is in progress (waiting on AWS to validate our request for an SSL certificate).
